### PR TITLE
Feature landing page

### PR DESCRIPTION
**Pull request by stevenCsaunders**
Styled landing page with tailwind classes. Mobile first. 
- Changes to App.js were made to speed up styling of the landing component.
- Removed header component so I was able to see landing page on live server for layout purposes.

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -1,35 +1,35 @@
...
+	return (
+		<>
+			<main>
+				<Switch>
+					<Route path='/' exact>
+						{authedRedirect(LandingPage)}
+					</Route>
+					<Route path='/login'>{authedRedirect(LoginPage)}</Route>
+					<Route path='/register'>
+						{authedRedirect(RegistrationPage)}
```
Comment (SamBrieden) [inline src/components/App.js:24]: We need to create a eslint config to prevent these large changes

**Review by SamBrieden — APPROVED** [review]

**Comment by stevenCsaunders** [comment]
Sorry that took so long to get straightened out.  Appreciate the quick review @SamBrieden!

File: tailwind.config.js [hunk 2]
```diff
@@ -1,4 +1,32 @@
 module.exports = {
+<<<<<<< HEAD
```
Comment (AlexanderPuhl) [inline tailwind.config.js:2]: This is the problem.

File: tailwind.config.js [hunk 3]
```diff
@@ -1,4 +1,32 @@
...
+		},
+	},
+	variants: {
+		extend: {
+			backgroundColor: ['active'],
+		},
+	},
+	plugins: [],
+}
+=======
```
Comment (AlexanderPuhl) [inline tailwind.config.js:29]: This is also causing the app to crash.

Comment (stevenCsaunders) [reply tailwind.config.js]: My bad.  I've merged in the remote changes for the tailwind.config file. Let me know if I need to do anything else with this PR.

**Comment by AlexanderPuhl** [comment]
Merged Stevens PR of styling the landing page into the main branch.

**Pull request merged**